README.md

```diff
@@ -578,6 +578,10 @@ This controls the number of changes processed at once before being sent to the t
 
 This controls the delay in milliseconds between processing each batch of changes.
 
+#### Sql_Trigger_MaxChangesPerWorker
+
+This controls the max number of changes handled by a worker, used for scaling decision.
```

Comment:
Maybe we should expand on this one. Adding @Charles-Gagnon.

'This controls the upper limit on the number of pending changes in the user table that are allowed per application-worker. If the count of changes exceeds this limit, it may result in a scale out. The setting only applies for Azure Function Apps with runtime driven scaling enabled. See the [Scaling](#scaling) section of more information.'

Reply:
That updated text looks great to me!

(except it should be `See the Scaling section for more information`